engine/pkg/cmd/cli/cli_job.go

```diff
@@ -61,12 +60,13 @@ func (o *jobGeneralOptions) addFlags(cmd *cobra.Command) {
 // validate checks that the provided job options are valid.
 func (o *jobGeneralOptions) validate(ctx context.Context, cmd *cobra.Command) error {
 	if len(o.masterAddrs) == 0 {
-		return errors.ErrInvalidCliParameter.GenWithStack("master-addrs can't be nil")
+		o.masterAddrs = []string{"127.0.0.1:10240"}
+		log.Warn("master-addrs is nil, use default addr: 127.0.0.1:10240")
```

Comment:
`nil` is a programmatic term. It's better to use `The master address does not assign, ...`

Reply:
Good suggestion. Already fix it, PTAL.